cloud/services/security/keypair.go

```diff
@@ -19,10 +19,11 @@ package security
 import (
 	"fmt"
 
+	_nethttp "net/http"
```

Comment:
Why do we need to import that now? Can we remove it here?

Reply:
removed